### ISSUE TITLE
Firefox Not Displaying Favicon

### ISSUE THREAD

**Issue by LowLifeArcade** [margaritahumanitarian/helpafamily#210]
### Favicon Not Displaying in FIrefox 

Currently the favicon works for Chrome, Edge, and Safari. 
Firefox displays nothing in the tab.

**What did you expect to happen?**

**Comment by RedFox0x20** [comment]
Cause of problem -- Favicon links within Firefox:
```
<link href="/branding/favicons/favicon-32x32.png" rel="icon" sizes="32x32" type="image/png">
<link href="/branding/favicons/favicon-16x16.png" rel="icon" sizes="16x16" type="image/png">
<link href="/favicon.ico" rel="icon">
```
> GET https://helpafamily.margaritahumanitarian.org/branding/favicons/favicon-16x16.png

Firefox shows that it is attempting to `GET`  the `favicon-16x16.png` this file does not exist. A solution to this would be to remove all of the links to favicons that are not available from the [_document.js file on line 56](https://github.com/margaritahumanitarian/helpafamily/blob/0c6ae41c018dbdf6aa2b05c2ab5879cf5989c038/pages/_document.js#L56)

**Comment by mktSuWin** [comment]
Hi @RedFox0x20 and @LowLifeArcade , I am Su, first year CS student. May I work on this issue? Thanks.

**Comment by ozer619** [comment]
Geat!! @mktSuWin let us know if you need any help.

**Comment by LowLifeArcade** [comment]
Yes, absolutely @mktSuWin! And thank you for your input @RedFox0x20 🥳 I'm sure this will help a lot.

**Comment by LowLifeArcade** [comment]
Hi @mktSuWin. Just checking in on you with this. If you need any help please feel free to ask.
